init-odkx-sync-endpoint.py

```diff
@@ -32,11 +39,48 @@ def run_interactive_config():
     typer.echo("We'll need some information from you to get started though...")
     time.sleep(1)
     typer.echo("")
+
+    allow_default = typer.confirm("Do you wish to use default configuratilon?", default=True)
```

Comment:
I don't know if we need two separate questions (to use the cache or to use the default config). If you do not use the cache, what do you do instead? (Should likely use the default?)

Part of what makes this script difficult is that we cannot have a default https configuration that works for all end users, since we need to know the deployment domain to build a valid configuration.